components/eam/src/physics/p3/eam/micro_p3_utils.F90

```diff
@@ -98,6 +98,113 @@ subroutine micro_p3_utils_init(cpair,rair,rh2o,rhoh2o,mwh2o,mwdry,gravit,latvap,
     integer, intent(in)     :: iulog
     logical(btype), intent(in)     :: masterproc
 
+    ! logfile info
+    iulog_e3sm      = iulog
+    masterproc_e3sm = masterproc
+
+    ! mathematical/optimization constants
+    thrd  = 1._rtype/3._rtype
+    sxth  = 1._rtype/6._rtype 
+    pi_e3sm = pi
+    piov3 = pi*thrd
+    piov6 = pi*sxth
```

Comment:
If `pi` is a parameter, we can declare `piov3` and piov6` as parameters too.

Reply:
Done.